hikari/messages.py

```diff
@@ -619,7 +620,7 @@ class PartialMessage(snowflakes.Unique):
     `type` is `MessageType.REPLY` and `builtins.None`, the message was deleted.
     """
 
-    @property
+    @property  # TODO: this breaks from the new convention for cache methods
```

Comment:
This logic is kinda off since it doesn't distinguish between the guild_id being unknown and the guild_id being known to be None